fixes/net.brinkervii.grapejuice.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright [2022] [The Grapejuice Developers] -->
+<component type="desktop-application">
+	<id>net.brinkervii.grapejuice.desktop</id>
```

Comment:
```suggestion
	<id>net.brinkervii.grapejuice</id>
```

This is supposed to be the app-id.

Reply:
com.valvesoftware.Steam has the .desktop ending

Reply:
I repeat again.

This is supposed to be the app-id.

Reply:
Yes, I happened to see the flathub requirement, I'll fix this

Reply:
Do I need the <launchable> tag?

Reply:
it's useful.

Reply:
amended